llvm/lib/Target/BPF/BPFASpaceCastSimplifyPass.cpp

```diff
@@ -0,0 +1,92 @@
+//===------------ BPFIRPeephole.cpp - IR Peephole Transformation ----------===//
```

Comment:
nit

Reply:
My bad.
Thank you for taking a look at this MR, will push updates with fixes soon.